src/app/controllers/censusdata/create.js

```diff
@@ -9,70 +28,35 @@
  */
 angular.module('flightNodeApp')
     .controller('CensusDataCreateController', 
-    ['$scope',
-    function ($scope) {
+    ['$scope', 'authService', 'config', 'messenger', 'censusFormService',
+    function ($scope, authService, config, messenger, censusFormService) {
```

Comment:
Hm, maybe you're already moving that direction.... I see `censusFormService` here :smile: